### PR TITLE
Feature johannes int main branch

### PR DESCRIPTION
**Pull request by JohannesHufnagl**
Finished the feature work

### CODE REVIEW

File: FeatureBranchTestJohannes/Recipe_dibba_rotti.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Instructions
+ 
+## Soaking Lentils And Rice Rava
+1. Rinse urad dal a couple of times in water. 
+2. Drain all the water and then soak urad dal in a bowl with 1 cup water for 3 to 4 hours. 
+3. Take rice rava or idli rava in a large bowl. We will first rinse the rava. 
+4. Add water till it floats a few centimetres on top of the rava layer. Gently mix the rava with water. Let this mixture rest for some seconds. Gently discard the water from the top. Do this method once or twice. 
+5. Then add 1.5 cups water for soaking rice rava for 3 to 4 hours.
```
Comment (spipau) [inline FeatureBranchTestJohannes/Recipe_dibba_rotti.md]: I would change this to:

```suggestion
5. Then add 1.5 cups water for soaking rice Rava for 3 to 4 hours.
```

Comment (JohannesHufnagl) [reply FeatureBranchTestJohannes/Recipe_dibba_rotti.md]: Fixed Typo

Comment (spipau) [reply FeatureBranchTestJohannes/Recipe_dibba_rotti.md]: Looks good now, thanks 👍

**Comment by spipau** [comment]
Ready for merge 👍

**Pull request merged**